.github/workflows/blueskyfeedbot.yml

```
@@ -0,0 +1,36 @@
+name: FeedBot
+on:
+  schedule:
+    # This will run every 10 minutes
+    - cron: '*/10 * * * *'
+jobs:
+  rss-to-bluesky:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate cache key
+        uses: actions/github-script@v6
+        id: generate-key
+        with:
+          script: |
+            core.setOutput('cache-key', new Date().valueOf())
+      - name: Retrieve cache
+        uses: actions/cache@v4
+        with:
+          path: ${{ github.workspace }}/blueskyfeedbot
+          key: feed-cache-${{ steps.generate-key.outputs.cache-key }}
+          restore-keys: feed-cache-
+      - name: GitHub
+        uses: 'joschi/blueskyfeedbot@v1'
+        with:
+          # This is the RSS feed you want to publish
+          rss-feed: https://nf-co.re/blog/rss.xml
+          # Template of status posted to Bluesky (Handlebars)
+          template: '{{item.title}} {{item.link}} @nf-co.re #nextflow'
```

Do we want to put some newlines in here?

I'd probably write it something like this:

```suggestion
          template: |
              New blog post! {{item.title}}
              {{item.link}}
```

Personally I don't bother with suffixed hashtags like that much in Bluesky.

---

okay, let's go with that then.